### PR TITLE
Update README badges

### PR DESCRIPTION
**Pull request by quba42**
[noissue]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +27,24 @@ The most important places:
 [5]: https://pypi.org/project/pulp-deb/
 [6]: https://pypi.org/project/pulp-deb-client/
 [7]: https://rubygems.org/gems/pulp_deb_client
```
Comment (hstct) [inline README.md:29]: It's weird to split the links up like that. Can we just put them all at the bottom?

**Review by hstct — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (quba42) [reply README.md:29]: Sure. I guess to be fully consistent, we should probably move the status update badge links into this list at the bottom as well...

**Review by hstct — APPROVED** [review]

**Comment by quba42** [comment]
Squished the commits without changing anything.

**Comment by quba42** [comment]
Note that this README is also used on [pypi.org](https://pypi.org/project/pulp-deb/).

**Pull request merged**